lldb/test/Shell/Commands/command-options.test

```diff
@@ -0,0 +1,11 @@
+# RUN: echo quit | %lldb -o "log enable -x" \
```

Comment:
Can you write one of these tests where the wrong option isn't the first?  From the standpoint of "things you could get wrong", "only working for the first option" seems a likely failure mode.

Reply:
Thanks! Indeed, there's bug there.